darwin/dataset/utils.py

```diff
@@ -160,10 +173,26 @@ def get_classes(
     dataset_path = Path(dataset_path)
```

Comment:
Is there a way we could take the opportunity to separate this function for torch dataloaders and remote datasets and only change this for the dataloaders?  I assume that's what you need?

The `get_classes` function touches a lot of code, so changes like this might have far reaching consequences we don't see here, and Darwin py is not yet with a proper set of regression tests, so I'm maybe being overcautious, but.

Reply:
Ok, after discussion, this seems ok.